File: sample/04_cgi/www/cgi/no_exec_permission.py
#!/usr/bin/python3

import sys

def main():
	# このスクリプトは smoke で実行権限を外し、403 を確認するためのダミー。
	# もし実行されたとしても、通常の CGI レスポンスを返す。
	sys.stdout.write("Content-Type: text/plain\r\n\r\n")
	sys.stdout.write("no_exec_permission.py\n")

if __name__ == "__main__":
	main()

File: sample/04_cgi/www/cgi/no_exec_permission.js
#!/usr/bin/node

// このスクリプトは smoke で実行権限を外し、403 を確認するためのダミー。
// もし実行されたとしても、通常の CGI レスポンスを返す。
process.stdout.write("Content-Type: text/plain\r\n\r\n");
process.stdout.write("no_exec_permission.js\n");
